File: Weekend Exercises/Python Version/task_thirteen.py
sentence = input("Enter a sentence: ").strip()

count = 1

for value in range(len(sentence)):

	if (sentence[value] == " "):
		count += 1

	if (sentence[value] == " " and sentence[value + 1] == " "):

		count -= 1

print("The number of words is", count)

File: Weekend Exercises/Java Version/TaskThirteen.java
import java.util.Scanner;
public class TaskThirteen{

	public static void main (String [] args){

		Scanner input = new Scanner(System.in);

		System.out.println("Enter a sentence: ");
		String sentence = input.nextLine().trim();

		int numberOfWords = 1;

		for (int index = 0; index < sentence.length(); index++){
			if(sentence.charAt(index) == ' '){
				numberOfWords++;
			}

			if(sentence.charAt(index) == ' ' && sentence.charAt(index+1) == ' '){
				numberOfWords--;
			}

		}

		System.out.printf("The number of words in %s is %d%n", sentence, numberOfWords);
	}

}